web/src/test/java/org/springframework/security/web/util/matcher/AntPathRequestMatcherTests.java

```diff
@@ -196,6 +197,13 @@ public void matchesWithInvalidMethod() {
 		assertThat(matcher.matches(request)).isFalse();
 	}
 
+	@Test
+	public void matcherWhenMatchAllPatternThenMatchResult() {
```

Comment:
Since this is a bug, will you please add `// gh-9285` above the test?

Reply:
Done.